src/components/content-padding.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+const contentPaddingStyle = StyleSheet.create({
+    container: {
+        height: '100%',
+        width: '100%',
+        padding: '5%',
```

Comment:
I think that flex: 1 is used instead of `height: 100%` and `width: 100%` in react native. I am not sure but I think that it is better to have a standard unit for padding instead of percent.